launch.md

```diff
@@ -22,7 +22,9 @@ Set or change the following options to control VS Code's behavior during debuggi
   Tells _GDB or LLDB_ what paths to search for .so files. Separate multiple paths with a semicolon. Example: `"/Users/user/dir1;/Users/user/dir2"`.
 
 * #### `externalConsole`
-  If set to `true`, launches an external console for the application. If `false`, no console is launched and VS Code's debugging console is used. Note this option is ignored in some cases for technical reasons.
+  Windows: When set to true, it will spawn an external console. When set to false, it will use VS Code's integratedTerminal.
+  Linux: When set to true, it will notify VS Code to spawn an external console. When set to false, it will use VS Code's integratedTerminal
```

Comment:
This line is missing a period. I fixed it in https://github.com/Microsoft/vscode-cpptools/pull/2750, along with other minor issues.